criteria/open-standards.md

```diff
@@ -9,9 +9,9 @@ order: 10
 * For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the [Open Source Initiative Open Standard Requirements](https://opensource.org/osr)
 * If no existing open standard is available, effort SHOULD be put into developing one
 * Standards that are machine testable SHOULD be preferred over those that are not
-* Functionality using features from a non-open standard (one that doesn't meet the [Open Source Initiative Open Standard Requirements](https://opensource.org/osr)) MAY be provided if necessary, but only in addition to compliant features.
+* Functionality using features from a non-open standard (one that doesn't meet the [Open Source Initiative Open Standard Requirements](https://opensource.org/osr)) MAY be provided if necessary, but only in addition to compliant features
 * All non-compliant standards used MUST be documented clearly in the documentation
-* The codebase MAY contain a list of all the standards used with links to where they are available
+* The codebase SHOULD contain a list of all the standards used with links to where they are available
```

Comment:
It's significantly easier to test for this criterion if there's a list in the documentation. Is there any reason not to make this MUST?

Reply:
I've made new issue https://github.com/publiccodenet/standard/issues/238 for this so this doesn't block the merge.